Automattic/wp-calypso
# Update Sandbox secret that changed recently

#### Changes proposed in this Pull Request

* The sandbox secret was changed. Updating the encrypted config file with the new value

#### Testing instructions

* Decrypt the new config using `npm run decryptconfig` and ensure that the value for `storeSandboxCookieValue` has been updated to match r196992-wpcom
